ScoutSuite/providers/aws/services/cloudwatch.py

```diff
@@ -28,12 +28,11 @@ def parse_alarm(self, global_params, region, alarm):
         alarm['name'] = alarm.pop('AlarmName')
         # Drop some data
         for k in ['AlarmConfigurationUpdatedTimestamp', 'StateReason', 'StateReasonData', 'StateUpdatedTimestamp']:
-            foo = alarm.pop(k) if k in alarm else None
+            alarm.pop(k) if k in alarm else None
```

Comment:
```suggestion
             if k in alarm:
                 alarm.pop(k)
```
Since there is no actual assignment done, I believe it is cleaner to use a normal if statement.